style.css

```diff
@@ -71,5 +71,237 @@ h1 {
   background-image: url(image/image_geometry_2.jpg);
   background-repeat: no-repeat;
   background-size: 100%;
-  height: 610px;
+  height: 260px;
+}
+
+h2 {
+  font-family: 'Playfair Display', sans-serif;
+  margin-left: 5%;
+  font-size: 50px;
+}
+
+.worksection1 {
+  background-image: url(image/Rectangle21.jpg);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 260px;
+  display: grid;
+}
+
+.worksection2 {
+  background-image: url(image/Rectangle22.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 260px;
+  display: grid;
+}
+
+.worksection3 {
+  background-image: url(image/Rectangle23.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 260px;
+  display: grid;
+}
+
+.worksection4 {
+  background-image: url(image/Rectangle24.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 260px;
+  display: grid;
+}
+
+.worksection5 {
+  background-image: url(image/Rectangle25.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 260px;
+  display: grid;
+}
+
+.rectangle {
+  background-color: #1a2236;
+  height: 156px;
+  display: grid;
+}
```

Comment:
- [ ] The cards are quite different than the Figma template. Consider using flexbox to style them. 😄 

![image](https://user-images.githubusercontent.com/92215878/191580797-de58204d-15e9-4609-8dcc-23483b4970cb.png)